client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cssSafeUrl from 'lib/css-safe-url';
+
+export class ThemesBanner extends Component {
+	static propTypes = {
+		title: PropTypes.string.isRequired,
+		description: PropTypes.string.isRequired,
+		ctaLabel: PropTypes.string.isRequired,
+		ctaAction: PropTypes.func,
+		backgroundImage: PropTypes.string,
+	};
+
+	renderButton = ( label, action, primary ) => {
+		return (
+			<button
+				type="button"
+				className={ primary ? 'button is-primary is-compact' : 'button is-compact' }
+				onClick={ action }
+			>
```

Comment:
Use Calypso's own [`Button` component](https://github.com/Automattic/wp-calypso/tree/master/client/components/button) (from `components/button`) instead:

```js
<Button
	primary
	compact
	onClick={ action }
/>
```

It also understands an `href` prop (see https://github.com/Automattic/wp-calypso/pull/20175/files#r159306126)